terraform/environments/nomis/nomis-test.tf

```diff
@@ -120,15 +120,15 @@ locals {
     }
 
     ec2_test_instances = {
-      t1-ndh-app = {
+      t1-ndh-app1 = {
```

Comment:
Just being pedantic, if you put t1-ndh-app-1 instead that follows convention elsewhere